relay-event-schema/src/protocol/span/convert.rs

```diff
@@ -63,17 +66,20 @@ macro_rules! context_value (
 );
 
 macro_rules! event_value(
+    ($event:expr, contexts browser $context_field:ident) => {
+        context_value!($event, BrowserContext, $context_field)
+    };
     ($event:expr, contexts trace $context_field:ident) => {
         context_value!($event, TraceContext, $context_field)
     };
     ($event:expr, contexts profile $context_field:ident) => {
         context_value!($event, ProfileContext, $context_field)
     };
-    ($event:expr, $path_root:ident $(. $path_segment:ident)*) => {
+    ($event:expr, $path_root:ident $($path_segment:ident)*) => {
         {
             let value = ($event).$path_root.value();
             $(
-                let value = value.and_then(|value|&value.$path_segment.value());
+                let value = value.and_then(|value|value.$path_segment.value());
```

Comment:
Turned out I had some unused code branches here that needed to be fixed for deep access.